### DISCUSSION TITLE
Script : Remove Different Lines?

### DISCUSSION THREAD

**Discussion by meteorquake** [WinMerge/winmerge#2551] [General]
I can see under Scripts there's "Remove Duplicate Lines", but there's no "Remove Different Lines" (i.e. leaving only duplicate lines), which is what I am after.
Am I missing it, or otherwise can I suggest it be added? It's actually a reasonable differencing task to want to list just the duplicates, as you may need to deal with them by pasting them into something else.
David

**Comment by sdottaka** [comment]
You can delete the lines with differences as follows:
1. Select the View → Diff Context → 0 Lines menu item. This will show only the differences.
2. Select all lines with Ctrl+A and press the Del key to delete them. This will delete only the differences.
3. Select the View → Diff Context → All Lines menu item to restore the original state.

![image](https://github.com/user-attachments/assets/f815b844-c493-4941-a70e-a3472a142a75)

**Comment by meteorquake** [comment]
Aha many thanks - that seems to work doesn't it! - D
